src/components/OverflowMenu/README.md

```diff
@@ -49,3 +49,11 @@ There are two important React props:
 - `primaryFocus` in `OverflowMenuItem`: This is required for the menu item you put keyboard focus on when `OverflowMenu` gets open
 
 Please refer to [our Storybook](http://react.carbondesignsystem.com/?selectedKind=OverflowMenu&selectedStory=basic) for more details.
+
+## Note about `<OverflowMenu>` children
+
+Make sure children of `<OverflowMenu>` are React components that accepts `ref` as its children - Typically `<OverflowMenuItem>`. Otherwise, you'll get an error like:
```

Comment:
```suggestion
Make sure the children of `<OverflowMenu>` are React components that accept `ref` as their children - Typically `<OverflowMenuItem>`. Otherwise, you'll get an error like:
```

Reply:
Just a little clearer -- kinda a nit, but thought I'd suggest 👍🏽

Reply:
Thanks @dakahn for your suggestion!